src/Stratis.Features.FederatedPeg/FedMultiSigWithdrawalTransactionBuilder.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NBitcoin;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Features.Wallet.Interfaces;
+using Stratis.Features.FederatedPeg.Interfaces;
+using Stratis.Features.FederatedPeg.Wallet;
+using Recipient = Stratis.Features.FederatedPeg.Wallet.Recipient;
+
+namespace Stratis.Features.FederatedPeg
+{
+    /// <summary>
+    /// Uses this node's federation wallet and the current network fee policy to build withdrawal transactions for the federation multisig.
+    /// </summary>
+    public class FedMultiSigWithdrawalTransactionBuilder
+    {
+        private readonly IFederationWalletManager federationWalletManager;
+
+        private readonly IFederatedPegSettings federatedPegSettings;
+        
+        private readonly Network network;
+
+        private readonly IWalletFeePolicy walletFeePolicy;
+
+        public FedMultiSigWithdrawalTransactionBuilder(
```

Comment:
We already have a `WithdrawalTransactionBuilder`, maybe put something in the name to suggest this is for manual building and signing?

Reply:
I agree, the naming should distinguish between the two more.